R/helpers.R

```diff
@@ -50,15 +91,12 @@
 .resolveConflicts <- function(name, indexes, dataset) {
   index <- indexes
   if (length(indexes) == 0) {
-    stop(paste0("There is no city called '", name, "' in the available data.\nUse 'new_city()' or create an issue including lat/long coordinates at https://github.com/koenderks/rcityviews/issues."))
+    stop(paste0("There is no city called '", name, "' in the available data.\n Use the 'new_city()' function to geocode the location"))
   } else if (length(indexes) > 1) {
     selection <- utils::menu(
       choices = paste0(dataset[indexes, 1], ", ", dataset[indexes, 2], " | Lat: ", round(dataset[indexes, 3], 3), " | Long: ", round(dataset[indexes, 4], 3)),
       title = "More than one city matched to this name, which one to pick?"
     )
-    if (selection == 0) {
```

Comment:
Revert these changes

Reply:
I prefer to work with `new_city` rather than any fallback